### PR TITLE
[MRG] Update the CHANGELOG

### PR DESCRIPTION
**Pull request by betatim**
Include recently merged changes in the changelog. Anything else to add?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,16 +6,20 @@ High five!
 
 ## New features
 
-* Single element dimension definition, which can be used to 
+* Single element dimension definition, which can be used to
 fix the value of a dimension during optimization.
-* `total_iterations` function of `BayesSearchCV` that 
-counts total iterations needed to explore all subspaces. 
-* Add iteration event handler for `BayesSearchCV`, useful 
+* `total_iterations` function of `BayesSearchCV` that
```
Comment (iaroslav-ai) [inline CHANGELOG.md]: Oops it is not actually a function it is a `@property`

**Comment by iaroslav-ai** [comment]
LGTM! (beyond the function -> property edit)

**Comment by iaroslav-ai** [comment]
Hm seems an issue with inverse transformation of samples from search space is what makes Travis upset
Could it be due to new clipping with "log-uniform"?
The error reads
```bash
@pytest.mark.fast_test
    def test_space_api():
        space = Space([(0.0, 1.0), (-5, 5),
                       ("a", "b", "c"), (1.0, 5.0, "log-uniform"), ("e", "f")])
    
        cat_space = Space([(1, "r"), (1.0, "r")])
        assert isinstance(cat_space.dimensions[0], Categorical)
        assert isinstance(cat_space.dimensions[1], Categorical)
    
        assert_equal(len(space.dimensions), 5)
        assert_true(isinstance(space.dimensions[0], Real))
        assert_true(isinstance(space.dimensions[1], Integer))
        assert_true(isinstance(space.dimensions[2], Categorical))
        assert_true(isinstance(space.dimensions[3], Real))
        assert_true(isinstance(space.dimensions[4], Categorical))
    
        samples = space.rvs(n_samples=10, random_state=0)
        assert_equal(len(samples), 10)
        assert_equal(len(samples[0]), 5)
    
        assert_true(isinstance(samples, list))
        for n in range(4):
            assert_true(isinstance(samples[n], list))
    
        assert_true(isinstance(samples[0][0], numbers.Real))
        assert_true(isinstance(samples[0][1], numbers.Integral))
        assert_true(isinstance(samples[0][2], str))
        assert_true(isinstance(samples[0][3], numbers.Real))
        assert_true(isinstance(samples[0][4], str))
    
        samples_transformed = space.transform(samples)
        assert_equal(samples_transformed.shape[0], len(samples))
        assert_equal(samples_transformed.shape[1], 1 + 1 + 3 + 1 + 1)
>       assert_array_equal(samples, space.inverse_transform(samples_transformed))
E       AssertionError: 
E       Arrays are not equal
E       
E       (mismatch 2.0%)
E        x: array([['0.5488135039273249', '1', 'a', '1.2595042808181094', 'f'],
E              ['0.7151893663724196', '2', 'c', '4.573990276587759', 'f'],
E              ['0.602763376071644', '2', 'c', '2.3160947584241445', 'f'],...
E        y: array([['0.5488135039273249', '1', 'a', '1.2595042808181094', 'f'],
E              ['0.7151893663724196', '2', 'c', '4.573990276587759', 'f'],
E              ['0.602763376071644', '2', 'c', '2.3160947584241445', 'f'],...
```

**Comment by iaroslav-ai** [comment]
Ok now this is confusing. The test passes on my machine

**Comment by iaroslav-ai** [comment]
Any ideas why it might fail specifically with Travis?

File: CHANGELOG.md (hunk 1 above)
Comment (betatim) [reply CHANGELOG.md]: Fixed

**Comment by betatim** [comment]
Will investigate the travis problem. Also passes for me. I reckon it is a bug that appears randomly :-/

**Comment by iaroslav-ai** [comment]
Found the difference between two arrays
In the first one: 1.949109588604984
in the second one: 1.9491095886049838
Not sure why

File: skopt/tests/test_space.py [hunk 2]
```diff
@@ -278,7 +278,8 @@ def test_space_api():
     samples_transformed = space.transform(samples)
     assert_equal(samples_transformed.shape[0], len(samples))
     assert_equal(samples_transformed.shape[1], 1 + 1 + 3 + 1 + 1)
-    assert_array_equal(samples, space.inverse_transform(samples_transformed))
+    assert_array_almost_equal(samples,
+                              space.inverse_transform(samples_transformed))
```
Comment (betatim) [inline skopt/tests/test_space.py]: other round-trip'ing tests also use `almost_equal` which is the way to compare things that might contain floating point errors.

**Comment by betatim** [comment]
I will pay out the finders bounty of 10c to both of you ;)

Ready for merge (again).

**Comment by glouppe** [comment]
Travis remains grumpy :/ 

(Feel free to merge once this is green, changes are minor.)

**Comment by betatim** [comment]
Switched to doing the comparison "by hand" as mentioned in the comment. Does someone know how we could do this with a built in tool/`assert_X`? Otherwise good to go from my side.

**Pull request merged**